### ISSUE TITLE
import numpy as np

### ISSUE THREAD

**Issue by ved7** [ved7/Exploratory-Project#1]
import matplotlib.pyplot as plt
from sklearn.cluster import KMeans
data=[(5,15),(7,20),(4,12),(8,21),(10,34),(23,10),(2,34),(7,46),(8,23),(2,15),(25,5),(17,10),(10,52),(18,11),(14,24),(2,20),(7,44),(5,46),(2,73),(7,35)]
X=np.array(data)
plt.scatter(X[:,0],X[:,1],label='True Position')
<matplotlib.collections.PathCollection at 0x2a7e8eed4e0>

kmeans = KMeans(n_clusters=5)
kmeans.fit(X)
KMeans(algorithm='auto', copy_x=True, init='k-means++', max_iter=300,
    n_clusters=5, n_init=10, n_jobs=1, precompute_distances='auto',
    random_state=None, tol=0.0001, verbose=0)
#printing the centroids coordinates 
print(kmeans.cluster_centers_)
[[20.75        9.        ]
 [ 7.25       47.        ]
 [ 6.25       18.75      ]
 [ 2.         73.        ]
 [ 6.33333333 34.33333333]]
print(kmeans.labels_)
[2 2 2 2 4 0 4 1 2 2 0 0 1 0 2 2 1 1 3 4]
plt.scatter(X[:,0],X[:,1],c=kmeans.labels_,cmap='rainbow')
<matplotlib.collections.PathCollection at 0x2a7e9633898>
#just a comment
